### DISCUSSION TITLE
Welcome to chatgpt_cli Discussions!

### DISCUSSION THREAD

**Discussion by zawawiAI** [zawawiAI/chatgpt_cli#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by h2omag21** [comment]
i'm told by openAI that if I go to here to github there is a way to detect/check text that is written by AI... where is this found - do you know? 

As a teacher, you can use the open-source technologies I mentioned earlier to check if a piece of text or image was generated by OpenAI. Here are a few ways you can use these tools:

GPT-2 and GPT-3: You can use the pre-trained models provided by OpenAI on GitHub to generate text similar to the text you suspect was generated by OpenAI. By comparing the generated text to the suspected text, you can determine if they match.

DALL-E: You can use the pre-trained model provided by OpenAI on GitHub to generate images from text descriptions. If the image in question matches the generated image, it is likely that the image was generated by OpenAI.

OpenAI API: You can use the OpenAI API to generate text similar to the text you suspect was generated by OpenAI. By comparing the generated text to the suspected text, you can determine if they match.

Hugging Face's transformers library: You can use pre-trained models provided by the library and fine-tune them for your task, you can compare the text generated by the model with the text you suspect was generated by OpenAI.

It's important to remember that these models can be fine-tuned and used by other people, so the content may not always be produced by OpenAI, but it will give you a good indication.

**Reply by zawawiAI** [reply]
Perhaps u can try this link https://gptzero.me/ to solve your problem.
